readme.md

```diff
@@ -175,6 +175,38 @@ Default: `process.cwd()`
 
 [Inform iTerm2](https://www.iterm2.com/documentation-escape-codes.html) of the current directory to help semantic history and enable [Cmd-clicking relative paths](https://coderwall.com/p/b7e82q/quickly-open-files-in-iterm-with-cmd-click).
 
+### iTerm.annotation(message, options?)
+
```

Comment:
Can you link to the iTerm docs about this thing?

And include a screenshot here of how it looks like.

Reply:
where should the image live? project root?

Reply:
Just upload it to a comment here and then use that URL.